pkg/conf/conf_test.go

```diff
@@ -13,6 +13,7 @@ func TestConfSimple(t *testing.T) {
 	TargetPort = 2008
 	SendTimeoutSec = 7
 	OutConnTimeoutSec = 9
+	KeepAliveSec = 1
```

Comment:
It would be nice to have a value different from the default one in the test. This would be useful if someone later changes the test work on defaults.

Reply:
The default value is 15 seconds for keepAlive. I went ahead with a lower value because we want to maintain connection and it takes 3 failed keepAlives to terminate connection.

Reply:
https://github.com/bookingcom/nanotube/blob/f9fc89923238b57e045f40c9e6d064bf325a1fba/pkg/conf/conf.go#L70
sorry, I mean this one